data/boarding missions.txt

```diff
@@ -61,6 +61,160 @@ mission "Assisting Merchant (Large)"
 
 
 
+mission "Asteroid Course A"
+	assisting
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	destination "Earth"
+	on offer
+		conversation
+			`As you are approaching the <origin>, your radar sensors detect that the ship you are assisting is in the middle of a large asteroid's course! While in a normal flight this wouldn't cause much trouble, a ship without active shields and that already suffered heavy damages could be completely destroyed.`
+			branch unarmed
+				has "armament deterrence" == 0
+			choice
+				`	(Shoot towards the asteroid to destroy it.)`
+					goto fire
+				`	(Move your ship between the asteroid and the <origin>, then use your shields to protect it.)`
+					goto move
+				`	(Try to quickly repair the <origin>, so that it can move away from the asteroid's course.)`
+					goto repair
+				`	(Interrupt the docking maneuvers and fly away.)`
+					goto escape
+			label unarmed
+			choice
+				`	(Move your ship between the asteroid and the <origin>, then use your shields to protect it.)`
+					goto move
+				`	(Try to quickly repair the <origin>, so that it can move away from the asteroid's course.)`
+					goto repair
+				`	(Interrupt the docking maneuvers and fly away.)`
+			label escape
+			`	You turn your ship around and activate your thrusters, trying to get as far as you can from the disabled ship. A few seconds later, the asteroid tears the <origin> apart, killing all of its crew members.`
+				launch
+			label fire
+			`	The shots you fire quickly break the asteroid into pieces, which keep following the same course: before you can even try to destroy them, most have already hit the <origin>, which is now quickly depressurizing and losing the hull's integrity. There's not much you can do anymore, except to try moving to safe distance from it...`
```

Comment:
Split the first sentence into two; with the colon, it gets a bit too long for my taste.
Change "try moving to safe distance from it" to "try to move a safe distance from the exploding ship"

Reply:
done